src/TeamsDriverStationDict.elm

```diff
@@ -0,0 +1,16 @@
+module TeamsDriverStationDict exposing (..)
```

Comment:
This isn't a `Dict` - you can use one with [core/dict](https://package.elm-lang.org/packages/elm/core/latest/Dict) if that's really what you want.